Add buttons to add Emerald to MetaMask (#80)
oasisprotocol/docs: docs/general/developer-resources/emerald-paratime/README.mdx

### Mainnet

<button
  class="button button--primary margin-bottom--md"
  onClick={() =>
    window.ethereum.request({
      method: 'wallet_addEthereumChain',
      params: [
        {
          chainId: '0xa516',
          chainName: 'Emerald Paratime Mainnet',
          nativeCurrency: {
            name: 'Emerald Rose',
            symbol: 'ROSE',
            decimals: 18,
          },
          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
        },
      ],
    })
  }
>

>   Click here to add Emerald Mainnet to MetaMask or Brave Wallet automatically

Reviewer: ```suggestion
  Click here to register Emerald Mainnet to your MetaMask or Brave Wallet
```

  * Decimal: 42262
* Block explorer: [https://explorer.emerald.oasis.dev](https://explorer.emerald.oasis.dev)

<button

>   class="button button--secondary button--outline"

Author: @matevz to make button more noticeable, the easiest is to switch to one of these variants:

- current, button--secondary button--outline
![current_outline-secondary](https://user-images.githubusercontent.com/3758846/164054152-38d66f49-ba84-4c38-a029-51ec8f37506b.png)

- button--outline
![outline](https://user-images.githubusercontent.com/3758846/164054166-0a4ea4ae-3e2c-4939-a85b-f4d968046899.png)

- button--secondary
![secondary](https://user-images.githubusercontent.com/3758846/164054249-85e3d437-3a3f-4535-b18e-7da94c4a8e23.png)

- button--primary
![primary](https://user-images.githubusercontent.com/3758846/164054186-3dadc94e-e83e-4c9f-b59c-1c3731ef16d7.png)

Author: oh wait, button--outline doesn't work well when hovered

  onClick={() =>
    window.ethereum.request({
      method: 'wallet_addEthereumChain',
      params: [
        {
          chainId: '0xa516',
          chainName: 'Emerald Paratime Mainnet',
          nativeCurrency: {
            name: 'Emerald Rose',
            symbol: 'ROSE',
            decimals: 18,
          },
          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
        },
      ],
    })
  }
>

>   Add Emerald Mainnet to MetaMask

Author: @gparty Oh nice. In that case:
```suggestion
  Add Emerald Mainnet to MetaMask or Brave Wallet
```

>   Add Emerald Mainnet to MetaMask or Brave Wallet

Reviewer: I would put it more explicit. Same for testnet.

```suggestion
  Click here to add Emerald Mainnet to MetaMask or Brave Wallet
```

Reviewer: Even better, could you move the button to the top just below the Mainnet subtitle and change the label to

 ```suggestion
  Click here to add Emerald Mainnet to MetaMask or Brave Wallet automatically
```

So folks see the button before they start reading the actual chain ids, endpoints etc. (and already on the way to filling metamask fields).

Author: New preview

![image](https://user-images.githubusercontent.com/3758846/164056516-139f6881-a18d-4bd4-9335-a94fabadf319.png)

Author: How about: 
![image](https://user-images.githubusercontent.com/3758846/164069799-3f67fb42-94c7-476f-92c4-ad8e250f9870.png)

### Testnet

<button
  class="button button--primary margin-bottom--md"
  onClick={() =>
    window.ethereum.request({
      method: 'wallet_addEthereumChain',
      params: [
        {
          chainId: '0xa515',
          chainName: 'Emerald Paratime Testnet',
          nativeCurrency: {
            name: 'TEST',
            symbol: 'TEST',
            decimals: 18,
          },
          rpcUrls: ['https://testnet.emerald.oasis.dev/', 'wss://testnet.emerald.oasis.dev/ws'],
          blockExplorerUrls: ['https://testnet.explorer.emerald.oasis.dev'],
        },
      ],
    })
  }
>

>   Click here to add Emerald Testnet to MetaMask or Brave Wallet automatically

Reviewer: ```suggestion
  Click here to register Emerald Testnet to your MetaMask or Brave Wallet
```
